leaderboard/LeaderboardRow/index.js

```diff
@@ -74,8 +75,8 @@ module.exports = React.createClass({
     let props = this.props
     let data = props.data
     let symbol = data.amount.currency.symbol
-    let formattedRaised = numeric.money(symbol, data.amount.cents)
-    let formattedGoal = numeric.money(symbol, data.target_cents)
+    let formattedRaised = numeric.money(symbol, data.amount.cents, '0.00')
+    let formattedGoal = numeric.money(symbol, data.target_cents, '0.00')
```

Comment:
These will always be dollar values. The valueFormat may be km, m. so might be different.